server/app.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const middleware = require('./middleware/index');
```

Comment:
u can delete index because it is the default but it's ok

Reply:
ok